packages/expo-localization/android/src/main/java/expo/modules/localization/LocalizationModule.kt

```diff
@@ -39,6 +42,21 @@ class LocalizationModule : Module() {
     Function("getCalendars") {
       return@Function getCalendars()
     }
+
+    OnCreate {
+      if (appContext.reactContext != null) setRTLFromStringResources(appContext.reactContext!!)
```

Comment:
```suggestion
      appContext.let {
        setRTLFromStringResources(it)
      }
```
It's better to use the `let` function instead of doing force unwrapping using `!!`.